signbank.py

```diff
@@ -78,9 +78,10 @@ def parse_signbank_csv(filename):
 ##########################
 # Asset handling
 ##########################
-def fetch_gloss_asset_export_file(filename):
+def fetch_gloss_asset_export_file(filename, filters = {}):
     session = signbank_session()
-    video_response = session.get("%s/video/csv" % DEFAULT_SIGNBANK_HOST)
+
+    video_response = session.get("%s/video/csv" % DEFAULT_SIGNBANK_HOST, params=filters)
```

Comment:
**suggestion**: old-style formatting is [a bit clunky](https://docs.python.org/3/library/stdtypes.html?highlight=sprintf#printf-style-string-formatting), this might be better represented with [an f-string](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals): `f"{DEFAULT_SIGNBANK_HOST}/video/csv"`

Reply:
Agreed! This is a nice thing to improve. Fixed in 2527c91